tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "lib": ["esnext"],
+    "allowJs": true,
+    "outDir": "build",
```

Comment:
Should'nt the **outDir** be `dist`?
As the path is [`./dist/server.js`](https://github.com/Real-Dev-Squad/website-backend/blob/42a16c6f1fc88f8039b4bd2ab0b0fb6c550a8999/package.json#L6) in package.json.

Reply:
Yes yes, my bad here. Thanks for pointing this out